config.yaml

```diff
@@ -1,281 +1,428 @@
-  maintainers:
+  - baentsch
+  - bhess
+  - christianpaquin
+  - jschanck
+  - Martyrshot
+  - praveksharma
   - SWilson4
-  members:
-  - fcasal
-  - tob-scott-a
-- name: tsc
+  - vsoftco
+
+# oqs-provider Maintainers
+# https://github.com/open-quantum-safe/oqs-provider/blob/main/GOVERNANCE.md#maintainers-1
+- name: oqs-provider-maintainers
+  maintainers:
+  - baentsch
+# oqs-provider Committers
+# https://github.com/open-quantum-safe/oqs-provider/blob/main/GOVERNANCE.md#committers-1
+- name: oqs-provider-committers
   maintainers:
   - baentsch
-  - dstebila
   members:
-  - ashman-p
   - bhess
-  - brian-jarvis-aws
   - christianpaquin
   - thb-sb
-  - vsoftco
-- name: www
+
+# boringssl Maintainers
+# TODO: provide "source of truth"
+- name: boringssl-maintainers
   maintainers:
-  - crockeea
+  - pi-314159
+
+# openssh Release Managers
+# TODO: provide "source of truth"
+- name: openssh-release-managers
+  maintainers:
+  - dstebila
   members:
-  - christianpaquin
+  - baentsch
+  - geedo0
   - praveksharma
-  - SWilson4
+
+# liboqs-cpp Maintainers
+# TODO: provide "source of truth"
+- name: liboqs-cpp-maintainers
+  maintainers:
+  - vsoftco
+# liboqs-cpp Release Managers
+# TODO: provide "source of truth"
+- name: liboqs-cpp-release-managers
+  maintainers:
+  - vsoftco
+
+# liboqs-go Maintainers
+# TODO: provide "source of truth"
+- name: liboqs-go-maintainers
+  maintainers:
+  - vsoftco
+# liboqs-go Release Managers
+# TODO: provide "source of truth"
+- name: liboqs-go-release-managers
+  maintainers:
+  - vsoftco
+
+# liboqs-python Maintainers
+# TODO: provide "source of truth"
+- name: liboqs-python-maintainers
+  maintainers:
+  - vsoftco
+# liboqs-python Release Managers
+# TODO: provide "source of truth"
+- name: liboqs-python-release-managers
+  maintainers:
   - vsoftco
+
+# liboqs-java Maintainers
+# TODO: provide "source of truth"
+- name: liboqs-java-maintainers
+  maintainers:
+  - dstebila
+  members:
+  - jimouris
+
+# liboqs-rust Maintainers
+# TODO: provide "source of truth"
+- name: liboqs-rust-maintainers
+  maintainers:
+  - thomwiggers
+# liboqs-rust Release Managers
+# TODO: provide "source of truth"
+- name: liboqs-rust-release-managers
+  maintainers:
+  - thomwiggers
+
+# The following is a private project intentionally hidden from view.
+# Contact praveksharma for details.
+- name: liboqs-cupqc-maintainers
+  maintainers:
+   - praveksharma
+  members:
+   - ydoroz
+   - stevenireeves
+   - neil-lindquist
+
 repositories:
 - name: .github
   teams:
-    core: read
+    oqs-admins: admin
+    tsc: read
+  visibility: public
+
+- name: tsc
+  teams:
+    oqs-admins: admin
     tsc: write
+    minute-takers: write
   visibility: public
-- name: boringssl
-  collaborators:
-    claucece: write
-    pi-314159: write
+
+- name: liboqs
   teams:
-    boringssl-maintainers: write
+    oqs-admins: admin
+    liboqs-maintainers: admin
+    oqs-release-managers: maintain
+    liboqs-committers: write
```

Comment:
It seems like the permissions provided to `<projectname>-committers` is inconsistent. In liboqs, the `liboqs-committers` team gets `write`, but below in oqs-providers, the `oqs-providers-committers` team gets `maintain`.  Is that intentional?

Reply:
That's an oversight on my part. I intended to give all Committers teams write access.

Reply:
Fixed in [236ba1c](https://github.com/open-quantum-safe/tsc/pull/77/commits/236ba1ce1b575dc1c73a8fea96dd0ef46b6a5762).